.circleci/config.yml

```diff
@@ -28,6 +28,11 @@ version: 2
           [[ "$CIRCLE_PR_NUMBER" == "" ]] && ./FTL-client "${FOLDER}" "${BIN_NAME}.sha1" "${FTL_SECRET}"
           rm ./FTL-client
           ls -lah .
+    - run:
```

Comment:
In the upload step, can this be added as the first command?
```
[ -z "$FTL_SECRET" ] && exit 0
```

This would make local testing via `circleci local execute --job x86_64` easier, as the step would not fail when uploading. This is already used in the API's Circle config.

(I commented on this line because GitHub won't let me comment on the upload step)

Reply:
Would this require the Test step to be run before the Upload step?

Reply:
No, this command will just skip the upload when running locally, as the secret is missing.

Reply:
Also, as we rename the binary, the `mv pihole-FTL "${BIN_NAME}"` command will have to be before this command.